wolfcrypt/src/evp.c

```diff
@@ -8788,3 +8853,4 @@ int  wolfSSL_EVP_DecodeFinal(WOLFSSL_EVP_ENCODE_CTX* ctx,
 #endif /* OPENSSL_EXTRA || OPENSSL_EXTRA_X509_SMALL */
 
 #endif /* WOLFSSL_EVP_INCLUDED */
+     
```

Comment:
Remove this extra line.

Reply:
Removed.

Reply:
I misspoke. I meant to leave the new line but remove the spaces on this line.

Reply:
Left back the new line there.